src/azure-cli/azure/cli/command_modules/vm/_params.py

```diff
@@ -385,6 +385,19 @@ def load_arguments(self, _):
     with self.argument_context('vm diagnostics set') as c:
         c.argument('storage_account', completer=get_resource_name_completion_list('Microsoft.Storage/storageAccounts'))
 
+    with self.argument_context('vm install-patches') as c:
+        c.argument('vm_name', arg_type=existing_vm_name)
+        c.argument('maximum_duration', type=str, help='Specifies the maximum amount of time that the operation will run. It must be an ISO 8601-compliant duration string such as PT4H (4 hours)')
+        c.argument('reboot_setting', type=str, help='Defines when it is acceptable to reboot a VM during a software update operation. Possible values are: IfRequired/ Never/Always')
+        c.argument('classifications_to_include', nargs='+', help='Space-separated list of classifications to include. Possible values for Windows VM: Critical/Security/UpdateRollUp/FeaturePack/'
+                                                                 'ServicePack/Definition/Tools/Updates. '
+                                                                 'Possible values for Linux VM: Critical/Security/Security')
+        c.argument('kb_numbers_to_include', nargs='+', help='Space-separated list of KBs to include in the patch operation. Applicable to Windows VM only')
+        c.argument('kb_numbers_to_exclude', nargs='+', help='Space-separated list of KBs to exclude in the patch operation. Applicable to Windows VM only')
+        c.argument('exclude_kbs_requiring_reboot', arg_type=get_three_state_flag(), help="Filters out KBs that don't have a reboot behavior of 'NeverReboots' when this is set. Applicable to Windows VM only")
+        c.argument('package_name_masks_to_include', nargs='+', help='Space-separated list of packages to include in the patch operation. Format: packageName_packageVersion. Applicable to Linux VM only')
+        c.argument('package_name_masks_to_exclude', nargs='+', help='Space-separated list of packages to exclude in the patch operation. Format: packageName_packageVersion. Applicable to Linux VM only')
```

Comment:
Any shorter names?

Reply:
origin long name has better readability as we discussed